WalletWasabi.Fluent/Helpers/TransactionFeeHelper.cs

```diff
@@ -74,6 +74,19 @@ public static bool TryEstimateConfirmationTime(Wallet wallet, FeeRate feeRate, [
 		return estimate is not null;
 	}
 
+	public static bool TryEstimateConfirmationTimeWithFeeAndVsize(Wallet wallet, int txFeeInSatoshi, int vSize, [NotNullWhen(true)] out TimeSpan? estimate)
```

Comment:
It's not used yet, but when I tested this feature, this was helpful, so I pushed it. It can be changed or removed if requested.